orchestra/utils/notifications.py

```diff
@@ -165,7 +164,6 @@ def _notify_experts_slack_status_change(task, current_worker):
 
 
 @run_if('ORCHESTRA_SLACK_EXPERTS_ENABLED')
-def message_experts_slack_group(slack_channel, text):
+def message_experts_slack_group(slack_channel, slack_message):
     slack = OrchestraSlackService(settings.SLACK_EXPERTS_API_KEY)
-    slack_message = format_slack_message(text)
```

Comment:
what about https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/communication/staffing.py#L85 and https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/communication/staffing.py#L130? Don't we need the message formatting still? how did this cause the `response_type` bug?